client/src/lib/dependency-injection/graphql/helpers.js

```diff
@@ -55,7 +58,14 @@ export const getFields = ({ args, fields, pagination = true }, stack = [ROOT_FIE
     const key = path.join('/');
     const fieldParams = args[key] || {};
 
-    const str = `${field}${getParams({ params: fieldParams, pagination: false })}`;
+    const items = (pagination) ? { ...fieldParams, ...paginationParams } : fieldParams;
+    const paramList = Object.entries(items)
+      .map(([paramName, varName]) => (
+        `${paramName}: $${varName}`
+      ));
+    const paramOutput = paramList.length ? `(${paramList.join(', ')})` : '';
```

Comment:
I'm not sure I'm a big fan of returning this list (and other similar lists) with the brackets included. It would be cleaner if the brackets were explicitly part of the place where the list is used and the list only returned the list, e.g. `readOne${getSingularName}(${getIdOnlyParams})` or as it was `${getQueryName}(${getParams})`.

What's your view on this?

Reply:
There are two reasons to include the brackets here:
1. That's how it already works and I want to avoid breaking things while I'm fixing things
2. Including the brackets here means we don't end up with empty brackets in the query if there are no fields/params/variables/whatever to return from these functions.